ajax/libs/fastify/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "fastify",
+  "title": "fastify",
+  "description": "Fast and low overhead web framework, for Node.js",
+  "author": {
+    "name": "Matteo Collina",
+    "url": "https://github.com/mcollina"
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/fastify/fastify.git"
+  },
+  "homepage": "https://www.fastify.io/",
+  "npmName": "fastify",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "**/*"
+      ]
```

Comment:
I don't believe we need to include the entire package here...

<details>
<summary><code>tree node_modules/fastify -> 24 directories, 271 files</code></summary>
<pre><code>
node_modules/fastify
├── LICENSE
├── README.md
├── SECURITY.md
├── build
│   └── build-validation.js
├── docs
│   ├── Benchmarking.md
│   ├── ContentTypeParser.md
│   ├── Decorators.md
│   ├── Ecosystem.md
│   ├── Errors.md
│   ├── Fluent-Schema.md
│   ├── Getting-Started.md
│   ├── HTTP2.md
│   ├── Hooks.md
│   ├── LTS.md
│   ├── Lifecycle.md
│   ├── Logging.md
│   ├── Middleware.md
│   ├── Plugins-Guide.md
│   ├── Plugins.md
│   ├── Recommendations.md
│   ├── Reply.md
│   ├── Request.md
│   ├── Routes.md
│   ├── Server.md
│   ├── Serverless.md
│   ├── Testing.md
│   ├── TypeScript.md
│   ├── Validation-and-Serialization.md
│   └── Write-Plugin.md
├── examples
│   ├── asyncawait.js
│   ├── example.js
│   ├── hooks-benchmark-async-await.js
│   ├── hooks-benchmark.js
│   ├── hooks.js
│   ├── http2.js
│   ├── https.js
│   ├── middleware.js
│   ├── parser.js
│   ├── plugin.js
│   ├── route-prefix.js
│   ├── shared-schema.js
│   ├── simple.js
│   ├── typescript-server.ts
│   └── use-plugin.js
├── fastify.d.ts
├── fastify.js
├── lib
│   ├── configValidator.js
│   ├── contentTypeParser.js
│   ├── context.js
│   ├── decorate.js
│   ├── errors.js
│   ├── fourOhFour.js
│   ├── handleRequest.js
│   ├── hooks.js
│   ├── initialConfigValidation.js
│   ├── logger.js
│   ├── middleware.js
│   ├── pluginUtils.js
│   ├── reply.js
│   ├── reqIdGenFactory.js
│   ├── request.js
│   ├── route.js
│   ├── schemas.js
│   ├── server.js
│   ├── symbols.js
│   ├── validation.js
│   ├── warnings.js
│   └── wrapThenable.js
├── node_modules
│   ├── ajv
│   │   ├── LICENSE
│   │   ├── README.md
│   │   ├── dist
│   │   │   ├── ajv.bundle.js
│   │   │   ├── ajv.min.js
│   │   │   └── ajv.min.js.map
│   │   ├── lib
│   │   │   ├── ajv.d.ts
│   │   │   ├── ajv.js
│   │   │   ├── cache.js
│   │   │   ├── compile
│   │   │   │   ├── async.js
│   │   │   │   ├── equal.js
│   │   │   │   ├── error_classes.js
│   │   │   │   ├── formats.js
│   │   │   │   ├── index.js
│   │   │   │   ├── resolve.js
│   │   │   │   ├── rules.js
│   │   │   │   ├── schema_obj.js
│   │   │   │   ├── ucs2length.js
│   │   │   │   └── util.js
│   │   │   ├── data.js
│   │   │   ├── definition_schema.js
│   │   │   ├── dot
│   │   │   │   ├── _limit.jst
│   │   │   │   ├── _limitItems.jst
│   │   │   │   ├── _limitLength.jst
│   │   │   │   ├── _limitProperties.jst
│   │   │   │   ├── allOf.jst
│   │   │   │   ├── anyOf.jst
│   │   │   │   ├── coerce.def
│   │   │   │   ├── comment.jst
│   │   │   │   ├── const.jst
│   │   │   │   ├── contains.jst
│   │   │   │   ├── custom.jst
│   │   │   │   ├── defaults.def
│   │   │   │   ├── definitions.def
│   │   │   │   ├── dependencies.jst
│   │   │   │   ├── enum.jst
│   │   │   │   ├── errors.def
│   │   │   │   ├── format.jst
│   │   │   │   ├── if.jst
│   │   │   │   ├── items.jst
│   │   │   │   ├── missing.def
│   │   │   │   ├── multipleOf.jst
│   │   │   │   ├── not.jst
│   │   │   │   ├── oneOf.jst
│   │   │   │   ├── pattern.jst
│   │   │   │   ├── properties.jst
│   │   │   │   ├── propertyNames.jst
│   │   │   │   ├── ref.jst
│   │   │   │   ├── required.jst
│   │   │   │   ├── uniqueItems.jst
│   │   │   │   └── validate.jst
│   │   │   ├── dotjs
│   │   │   │   ├── README.md
│   │   │   │   ├── _limit.js
│   │   │   │   ├── _limitItems.js
│   │   │   │   ├── _limitLength.js
│   │   │   │   ├── _limitProperties.js
│   │   │   │   ├── allOf.js
│   │   │   │   ├── anyOf.js
│   │   │   │   ├── comment.js
│   │   │   │   ├── const.js
│   │   │   │   ├── contains.js
│   │   │   │   ├── custom.js
│   │   │   │   ├── dependencies.js
│   │   │   │   ├── enum.js
│   │   │   │   ├── format.js
│   │   │   │   ├── if.js
│   │   │   │   ├── index.js
│   │   │   │   ├── items.js
│   │   │   │   ├── multipleOf.js
│   │   │   │   ├── not.js
│   │   │   │   ├── oneOf.js
│   │   │   │   ├── pattern.js
│   │   │   │   ├── properties.js
│   │   │   │   ├── propertyNames.js
│   │   │   │   ├── ref.js
│   │   │   │   ├── required.js
│   │   │   │   ├── uniqueItems.js
│   │   │   │   └── validate.js
│   │   │   ├── keyword.js
│   │   │   └── refs
│   │   │       ├── data.json
│   │   │       ├── json-schema-draft-04.json
│   │   │       ├── json-schema-draft-06.json
│   │   │       ├── json-schema-draft-07.json
│   │   │       └── json-schema-secure.json
│   │   ├── package.json
│   │   └── scripts
│   │       ├── bundle.js
│   │       ├── compile-dots.js
│   │       ├── info
│   │       ├── prepare-tests
│   │       ├── publish-built-version
│   │       └── travis-gh-pages
│   ├── fast-deep-equal
│   │   ├── LICENSE
│   │   ├── README.md
│   │   ├── es6
│   │   │   ├── index.d.ts
│   │   │   ├── index.js
│   │   │   ├── react.d.ts
│   │   │   └── react.js
│   │   ├── index.d.ts
│   │   ├── index.js
│   │   ├── package.json
│   │   ├── react.d.ts
│   │   └── react.js
│   └── json-schema-traverse
│       ├── LICENSE
│       ├── README.md
│       ├── index.js
│       ├── package.json
│       └── spec
│           ├── fixtures
│           │   └── schema.js
│           └── index.spec.js
├── package.json
└── test
    ├── 404s.test.js
    ├── 500s.test.js
    ├── async-await.js
    ├── async-await.test.js
    ├── bodyLimit.test.js
    ├── case-insensitive.test.js
    ├── chainable.test.js
    ├── close-pipelining.test.js
    ├── close.test.js
    ├── content-length.test.js
    ├── context-config.test.js
    ├── custom-http-server.test.js
    ├── custom-parser-async.js
    ├── custom-parser.test.js
    ├── custom-querystring-parser.test.js
    ├── decorator.test.js
    ├── delete.test.js
    ├── emit-warning.test.js
    ├── error-in-post.test.js
    ├── esm
    │   ├── esm.mjs
    │   ├── index.test.js
    │   ├── other.mjs
    │   └── plugin.mjs
    ├── fastify-instance.test.js
    ├── fluent-schema.js
    ├── fluent-schema.test.js
    ├── genReqId.test.js
    ├── get.test.js
    ├── handler-context.test.js
    ├── head.test.js
    ├── helper.js
    ├── hooks-async.js
    ├── hooks.test.js
    ├── http2
    │   ├── closing.js
    │   ├── http2.test.js
    │   ├── missing-http2-module.js
    │   ├── plain.js
    │   ├── secure-with-fallback.js
    │   ├── secure.js
    │   └── unknown-http-method.js
    ├── https
    │   ├── custom-https-server.test.js
    │   ├── fastify.cert
    │   ├── fastify.key
    │   └── https.test.js
    ├── inject.test.js
    ├── input-validation.js
    ├── input-validation.test.js
    ├── internals
    │   ├── all.test.js
    │   ├── contentTypeParser.test.js
    │   ├── decorator.test.js
    │   ├── errors.test.js
    │   ├── handleRequest.test.js
    │   ├── hookRunner.test.js
    │   ├── hooks.test.js
    │   ├── initialConfig.test.js
    │   ├── logger.test.js
    │   ├── plugin.test.js
    │   ├── reply.test.js
    │   ├── schemas.test.js
    │   └── validation.test.js
    ├── listen.test.js
    ├── logger.test.js
    ├── middleware.test.js
    ├── nullable-validation.test.js
    ├── options.error-handler.test.js
    ├── options.test.js
    ├── output-validation.test.js
    ├── patch.error-handler.test.js
    ├── patch.test.js
    ├── plugin.helper.js
    ├── plugin.test.js
    ├── post.test.js
    ├── pretty-print.test.js
    ├── promises.test.js
    ├── proto-poisoning.test.js
    ├── put.error-handler.test.js
    ├── put.test.js
    ├── register.test.js
    ├── reply-error.test.js
    ├── request-error.test.js
    ├── route-hooks.test.js
    ├── route-prefix.test.js
    ├── route.test.js
    ├── router-options.test.js
    ├── schemas.test.js
    ├── shared-schemas.test.js
    ├── skip-reply-send.js
    ├── stream.test.js
    ├── throw.test.js
    ├── trust-proxy.test.js
    ├── types
    │   ├── index.ts
    │   └── tsconfig.json
    ├── validation-error-handling.test.js
    ├── versioned-routes.test.js
    └── wrapThenable.test.js

24 directories, 271 files
</code></pre>
</details>

Reply:
@mcollina could you help and let us know what directory/files should be published on cdnjs?

Reply:
fastify.js and all content of lib/ should be enough to run the framework.

However, isn’t cdnjs just for code that can run in a Browser?

Reply:
Ok, thanks. I'm not sure what's the intent to add it but if Node.js decides to supports URL as well, it could work I suppose.